core/src/main/scala/com/softwaremill/session/CsrfDirectives.scala

```diff
@@ -14,16 +14,20 @@ trait CsrfDirectives {
     * Note that this scheme can be broken when not all subdomains are protected or not using HTTPS and secure cookies,
     * and the token is placed in the request body (not in the header).
     *
+    * The risk of the previous disclaimer is mitigated by using a concatenation of a timestamp and its HMAC hash
```

Comment:
So it's no longer a problem when subdomains are out of the user's control? If so, would be good to just remove that comment and maybe add a link to the appropriate OWASP section?

Reply:
Added a reference to OWASP and moved my addition one paragraph up.